pkgs/os-specific/linux/lm-sensors/default.nix

```diff
@@ -7,22 +7,21 @@ assert sensord -> rrdtool != null;
 
 stdenv.mkDerivation rec {
   name = "lm-sensors-${version}";
-  version = "3.4.0"; # don't forget to tweak fedoraproject mirror URL hash
+  version = "3.5.0"; # don't forget to tweak fedoraproject mirror URL hash
 
   src = fetchurl {
     urls = [
       # "http://dl.lm-sensors.org/lm-sensors/releases/lm_sensors-${version}.tar.bz2" # dead
       # https://github.com/lm-sensors/lm-sensors/releases/... # only generated tarballs
-      "https://src.fedoraproject.org/repo/pkgs/lm_sensors/lm_sensors-${version}.tar.bz2/c03675ae9d43d60322110c679416901a/lm_sensors-${version}.tar.bz2"
+      #"https://src.fedoraproject.org/repo/pkgs/lm_sensors/lm_sensors-${version}.tar.bz2/c03675ae9d43d60322110c679416901a/lm_sensors-${version}.tar.bz2"
+      https://github.com/lm-sensors/lm-sensors/archive/V3-5-0.tar.gz # only generated tarballs
```

Comment:
That's the real upstream source from https://hwmon.wiki.kernel.org/lm_sensors and should come first
**But** this is not a stable URL for `fetchurl` (GitHub generated archive) so rather use `fetchzip` and compute the SHA hash over the unpacked content